src/components/ADempiere/Form/VPOS/KeyLayout/index.vue

```diff
@@ -64,7 +64,12 @@
                 </div>
                 <div class="footer-product">
                   <p class="quantity">
-                    Cantidad: {{ formatQuantity(keyValue.quantity) }}
+                    Cantidad:
```

Comment:
Just can see change for source code with the same functionality, please let it without changes